src/pages/authorization/agent-authorization/index.mdx

```diff
@@ -313,11 +313,11 @@ https://accounts.livechat.com/
 | Parameter               | Required | Required with PKCE | Description                                                                                                                                                                                                            |
 | ----------------------- | -------- | ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `response_type`         | yes      | yes                | Value: `code`                                                                                                                                                                                                          |
-| `client_id`             | yes      | yes                | **Client Id** from Developer Console (Authorization block)                                                                                                                                                             |
-| `redirect_uri`          | yes      | yes                | One of the URIs defined in the Authorization block during app configuration. The LiveChat OAuth Server will redirect the user back to this URI after successful authorization.                                         |
-| `state`                 | no       | no                 | Any value that might be useful to your application. It's strongly recommended to include an anti-forgery token to mitigate the [cross-site request forgery](https://en.wikipedia.org/wiki/Cross-site_request_forgery). |
-| `code_challenge`        | no       | yes                | A string between 43 and 128 characters long.                                                                                                                                                                           |
-| `code_challenge_method` | -        | no                 | Possible values: `s256` or `plain`; default: `plain`.                                                                                                                                                                  |
+| `client_id`             | yes      | yes                | **Client Id** from Developer Console (Authorization block)                                                                                          |
+| `redirect_uri`          | yes      | yes                | `redirect_uri` should be the same as `Direct installation URL` defined in the Authorization block during app configuration. The LiveChat OAuth Server will redirect the user back to this URI after successful authorization. |
```

Comment:
Fix based on the [case from discrod](https://discord.com/channels/751046617527746560/816251703669162014/1110446810742403152) and consultation with Michał Toporowski who was testing it.